llama_cpp/llama.py

```diff
@@ -759,6 +759,10 @@ def __init__(
         # Chat Format Params
         chat_format: str = "llama-2",
         chat_handler: Optional[llama_chat_format.LlamaChatCompletionHandler] = None,
+        # Cache
+        cache: bool = False,
+        cache_type: str = "ram",
+        cache_size: int = 2 << 30,
```

Comment:
Can we keep these cache changes out of this PR, I'm currently working on the cache as part of batch processing and want to avoid any conflicts.